tests/manage/z_cluster/test_ceph_default_values_check.py

```diff
@@ -0,0 +1,72 @@
+import logging
+import pytest
+
+from ocs_ci.framework.testlib import ManageTest, tier1
+from ocs_ci.ocs.resources import pod
+from ocs_ci.ocs.cluster import get_pg_balancer_status
+
+log = logging.getLogger(__name__)
+
+
+@tier1
+@pytest.mark.polarion_id("OCS-2231")
+class TestCephDefaultValuesCheck(ManageTest):
+
+    def test_ceph_default_values_check(self):
+        """
+        This test checks ceph default values taken from OCS 4.5 with the
```

Comment:
should OCS 4.3 be the minimum version in which these values apply?

Reply:
I think you are right. When full disk handling was introduced (4.3), the values were change to the values in this PR. Till then the values were not exposed to OCS and were ceph default (full ratio 95% nearfull 85% as far as I remember)

Reply:
yes, these values are exposed in OCS from 4.3.
I will replace the version in this comment to 4.3

Reply:
Done